index.js

```diff
@@ -531,6 +533,16 @@ function createPanZoom(domElement, options) {
       e.stopPropagation()
       return false;
     }
+
+    // if suppress function is provided, pan will start only if it
+    // evaluates to false
```

Comment:
should suppressPanForEvent default to noop if not provided ? My personal preference is to not get too defensive and let errors bubble if e.g. a non function obj is passed in as suppressPanForEvent. I guess we could add additional checking/error handling here.